yarl/_url.py

```diff
@@ -298,7 +299,7 @@ def __bytes__(self):
         return str(self).encode("ascii")
 
     def __eq__(self, other):
-        if not type(other) is URL:
+        if type(other) is not URL:
```

Comment:
I think this changes the semantics of the check so I wouldn't include it in this PR.

Reply:
This change is not made by me. It was added by you in a force-push activity:
![image](https://github.com/aio-libs/yarl/assets/314607/ad4fc304-b6d8-442c-9c4f-b08495da9c32)

Reply:
Nope: https://github.com/aio-libs/yarl/pull/898#discussion_r1414784046 / https://github.com/aio-libs/yarl/commit/e906337174579cb4d8bac5c1f44ec5ca280a7c3c.

Reply:
Sorry.